src/config.ts

```diff
@@ -0,0 +1,32 @@
+import * as dotenv from "dotenv";
+
+const config = dotenv.config().parsed!;
+
+Object.keys(config).forEach(key => {
+  process.env[key] = config[key];
+});
```

Comment:
```suggestion

if (config) {
  Object.keys(config).forEach(key => {
    process.env[key] = config[key];
  });
}
```
.env がないときも考慮したほうがよいです